cylc/flow/scripts/lint.py

```diff
@@ -1271,6 +1282,4 @@ def main(parser: COP, options: 'Values', target=None) -> None:
 
 # NOTE: use += so that this works with __import__
 # (docstring needed for `cylc help all` output)
-__doc__ += TOMLDOC.format(
-    '\n\n.. code-block:: toml\n', str(LINT_SECTIONS)) + get_reference_rst(
-    parse_checks(['728', 'style'], reference=True))
+__doc__ += get_reference('all', 'rst')
```

Comment:
Is this supposed to be
```python
__doc__ += TOMLDOC.format(
    '\n\n.. code-block:: toml\n', str(LINT_SECTIONS)
) + get_reference('all', 'rst')
```
?

Reply:
Actually I think not, this was what is currently causing cylc-doc build failures but your changes fix it

Reply:
(Will need a fix backported to 8.2.x)